cdc/scheduler/scheduler_dispatcher_test.go

```diff
@@ -0,0 +1,72 @@
+// Copyright 2021 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package scheduler
+
+import (
+	"github.com/pingcap/log"
+	"github.com/pingcap/ticdc/cdc/model"
+	cdcContext "github.com/pingcap/ticdc/pkg/context"
+	"github.com/stretchr/testify/mock"
+	"go.uber.org/zap"
+)
+
+type mockScheduleDispatcherCommunicator struct {
```

Comment:
seems the mock is modified after generated, any rules should fellow if someone else modify later

Reply:
This mock is not generated. Do you have any specific suggestions about the rules?